static/styles/portico-signin.scss

```diff
@@ -628,6 +628,16 @@ button.login-google-button {
     transform: translateX(15px) translateY(13px);
 }
 
+.azure-wrapper::before {
+    content: "\f3ca";
```

Comment:
This styling seems to not work; note that we're using version 4.7.0 of FontAwesome, since it's their last fully open source version.

![image](https://user-images.githubusercontent.com/2746074/46772507-23648980-ccad-11e8-9357-0f4b538c763b.png)